MAUI/Parallax-View/custom-controls.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title: Custom controls in .NET MAUI Parallax View Control | Syncfusion
+description: Learn here all about Scrolling support for custom controls support in Syncfusion .NET MAUI Parallax View (SfParallaxView) control and more.
```

Comment:
description: Learn about Scrolling support for custom controls support in Syncfusion .NET MAUI Parallax View (SfParallaxView) control and more.